### DISCUSSION TITLE
Stay in Prompt mode?

### DISCUSSION THREAD

**Discussion by PlayBoxTech** [Doggettx/stable-diffusion#7] [Q&A] [answered]
How difficult would it be to make yours able to stay in prompt mode? (as opposed to exiting after every image?)

**Comment by Doggettx** [comment, marked as answer]
This repo is more meant for example how to implement things. It's fairly easy to just use another repo that has the abilities you want and then copy over the parts you want from this one. For example the optimizations are all located in these 2 files:

ldm\modules\attention.py  
ldm\modules\diffusionmodules\model.py
